src/rustdoc.md

```diff
@@ -36,30 +36,174 @@ does is call the `main()` that's in this crate's `lib.rs`, though.)
 
 ## Cheat sheet
 
-* Use `./x.py build` to make a usable
-  rustdoc you can run on other projects.
+* Copy config.toml.example (in the root of your Rust checkout to config.toml,
+  add add the lines below. This will speed up build times dramatically by not
+  building rustc locally. And skipping minifcation will make debugging JS
+  easier.
+    download-rustc = true
+    download-ci-llvm = true
+    skip-rebuild = true
+    docs-minification = false
+* Use `./x.py build` to make a usable rustdoc you can run on other projects.
   * Add `library/test` to be able to use `rustdoc --test`.
-  * If you've used `rustup toolchain link local /path/to/build/$TARGET/stage1`
-    previously, then after the previous build command, `cargo +local doc` will
-    Just Work.
+  * Use `rustup toolchain link local /path/to/build/$TARGET/stage1`
+    to add a custom toolchain named `local`. So long as you have a built rustdoc,
+    you can run `cargo +local doc` to create docs using your locally-built
+    rustdoc.
 * Use `./x.py doc library/std` to use this rustdoc to generate the
   standard library docs.
-  * The completed docs will be available in `build/$TARGET/doc/std`, though the
-    bundle is meant to be used as though you would copy out the `doc` folder to
-    a web server, since that's where the CSS/JS and landing page are.
+  * The completed docs will be available in `build/$TARGET/doc/std`.
 * Use `./x.py test src/test/rustdoc*` to run the tests using a stage1 rustdoc.
   * See [Rustdoc internals] for more information about tests.
-* Most of the HTML printing code is in `html/format.rs` and `html/render.rs`.
+
+## Code structure
+
+* Most of the HTML printing code is in `src/librustdoc/html/format.rs` and
+  `src/librustdoc/html/render/mod.rs`.
   It's in a bunch of `fmt::Display` implementations and supplementary
   functions.
-* The types that got `Display` impls above are defined in `clean/mod.rs`, right
+* The types that got `Display` impls above are defined in `src/librustdoc/clean/mod.rs`, right
```

Comment:
Do we really need to prefix _everything_ with `src/librustdoc`? I'd rather add one paragraph at the beginning of the section saying that rustdoc is in `src/librustdoc`, and then use relative paths here.